util/app-config/src/args.rs

```diff
@@ -111,6 +112,23 @@ pub struct InitArgs {
     pub block_assembler_message: Option<String>,
     /// TODO(doc): @doitian
     pub import_spec: Option<String>,
+    /// Don't customize any parameters for chain spec, use the default parameters.
+    ///
+    /// Only works for dev chains.
+    pub use_default_spec: bool,
```

Comment:
I'm wondering why this parameter is needed, isn't it equivalent to `CustomizeSpec#is_unset`?